.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [Linux, macOS, Windows]
+        os: [Linux, macOS]
```

Comment:
Do we not care about windows anymore?

Reply:
The windows agent was out to lunch so I temporarily disabled it. I'll add it back to see if it completes.

Longer term, we need to get more robust windows machines.